invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/components/BookMetadata/BookMetadata.js

```diff
@@ -0,0 +1,274 @@
+import React, { Component } from 'react';
+import {
+  Grid,
+  Segment,
+  Image,
+  Responsive,
+  Container,
+  Button,
+  Header,
+  List,
+  Icon,
+} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { invenioConfig } from '../../../../../common/config';
+import BookTab from '../BookTab';
+import { RequestNewLoanForm } from './components/RequestNewLoanForm';
+import '../../BookDetails.scss';
+
+export default class BookMetadata extends Component {
+  state = { visible: false };
+
+  toggleVisibility = () => this.setState({ visible: !this.state.visible });
+
+  _render_book_info() {
+    const bookData = this.props.bookDetails;
+    return (
+      <div className="book-info">
+        <Header as="h2">{bookData.title}</Header>
+        <List>
+          {bookData.authors.map((author, index) => (
+            <List.Item as="h4" key={`Key${index}`}>
```

Comment:
Generally it's recommended to use something unique as the key that is not an index in case the array is reordered. If `author` is guaranteed to be unique then that's a good option. If you have nothing that uniquely describes each item, `index` can be used as a last resort.